velox/vector/VariantToVector.cpp

```diff
@@ -77,4 +77,158 @@ ArrayVectorPtr variantArrayToVector(
       pool);
 }
 
+struct NestedTypeCounter {
+  TypeKind kind = TypeKind::UNKNOWN;
+  vector_size_t rowCount = 0;
```

Comment:
nit: What does rowCount represent here?

Reply:
Sorry, it should better be named memberCount. It's used to keep count of the number of elements present in the vector.